protocol-tests/src/main/java/fixtures/url/QueriesAsyncClient.java

```diff
@@ -658,7 +658,7 @@
      * <table border="1">
      *     <caption>Query Parameters</caption>
      *     <tr><th>Name</th><th>Type</th><th>Required</th><th>Description</th></tr>
-     *     <tr><td>arrayQuery</td><td>String</td><td>No</td><td>a null array of string using the csv-array format</td></tr>
+     *     <tr><td>arrayQuery</td><td>String</td><td>No</td><td>a null array of string using the csv-array format In the form of "," separated string.</td></tr>
```

Comment:
You might check if you can put a ending period (dot) to the original description, if it is not empty and is not end with a period.